pkg/ingester/client/cortex.proto

```diff
@@ -122,3 +123,16 @@ message LabelMatcher {
   string name = 2;
   string value = 3;
 }
+
+message HealthCheckRequest {
+  string service = 1;
+}
+
+message HealthCheckResponse {
+  enum ServingStatus {
```

Comment:
Do UNKNOWN or NOT_SERVING ever get sent?

Reply:
They do not right now, that is the format suggested for health-checking by the gRPC docs. We could make a more cortex specific format if we want.

Reply:
It will also be compatible with: https://godoc.org/google.golang.org/grpc/health

I could also try to spend some time importing the proto definitions from grpc/health, rather than duplicating them.

Reply:
Yeah, I'd say either use those protos or make our response an empty response.